vue-frontend/src/components/contact/V2ContactForm.vue

```diff
@@ -0,0 +1,423 @@
+          <div class="tw-pt-2">
+            <input
+              class="custom-text-input v2-title-2-text"
+              type="text"
+              name="username"
+              required
+              autocomplete="given-username"
+              v-model="name"
+              :disabled="disableInput"
+              placeholder="Your name"
+            />
+            <span v-if="showValidationError" class="error tw-text-red-600"
+              >This field is required</span
+            >
+          </div>
+          <div class="lg:tw-pt-14">
+            <input
+              class="custom-text-input v2-title-2-text"
+              type="text"
+              name="email"
+              required
+              autocomplete="given-email"
+              v-model="email"
+              :disabled="disableInput"
+              placeholder="Your email"
+            />
+            <span v-if="showValidationError" class="error tw-text-red-600"
+              >This field is required</span
+            >
+          </div>
+          <div class="lg:tw-pt-14">
+            <textarea
+              class="custom-text-input v2-title-2-text"
+              name="project"
+              rows="1"
+              autocomplete="given-project-info"
+              v-model="projectInfo"
+              :disabled="disableInput"
+              placeholder="Tell us about your project"
+            ></textarea>
+          </div>
+          <div class="lg:tw-pt-10">
+            <input
+              class="custom-text-input v2-title-2-text"
+              type="text"
+              name="reference"
+              required
+              autocomplete="given-reference"
+              v-model="reference"
+              :disabled="disableInput"
+              placeholder="How did you find us?"
+            />
+            <span v-if="showValidationError" class="error tw-text-red-600"
+              >This field is required</span
+            >
+          </div>
+
+          <div class="tw-py-16">
+            <div class="v2-title-2-text">
+              What's your preferred mode of communication?
+            </div>
+            <div class="tw-flex tw-flex-col tw-w-fit sm:tw-flex-row tw-pt-6">
+              <label class="contactLabel1"
+                ><input
+                  class="tw-hidden"
+                  type="radio"
+                  name="contact_type"
+                  value="0"
+                  v-model.number="contactType"
+                  :disabled="disableInput"
+                />
+                <div
+                  class="contact-button tw-mb-6 sm:tw-mr-9 sm:tw-mb-0"
+                  :class="[
+                    contactType == 0
+                      ? 'contact-button-active'
+                      : 'contact-button',
+                  ]"
+                >
+                  <font-awesome-icon
+                    class="fa -tw-rotate-45 tw-w-6 tw-h-6 tw-text-black-900"
+                    :icon="phone"
```

Comment:
Icon is different then figma

Reply:
Because that same icon is not in font awesome